src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -25,6 +25,7 @@ object ErrorAggregator {
   private val allowedAppNames = List("Firefox")
   private val outputPrefix = "error_aggregates/v2"
   private val kafkaCacheMaxCapacity = 1000
+  private val numPartitions = 600
```

Comment:
Where does 600 come from?

Reply:
right now: 180GB total. 180 / 600 = ~300MB per file, a good size for parquet.

Obviously this will need to be updated when the job runs :)

Reply:
Do you mind to add that as a comment?